mpeg2/src/pes.rs

```diff
@@ -13,6 +15,7 @@ pub struct PacketizationConfig {
     pub packet_id: u16,
     pub random_access_indicator: bool,
     pub continuity_counter: u8,
+    pub temi: Option<Vec<TEMITimelineDescriptor>>,
```

Comment:
Also, for consistency:

```suggestion
    pub temi_timeline_descriptors: Vec<TEMITimelineDescriptor>,
```

Reply:
just lazy, but should make it consistent